src/Entity/Package.php

```diff
@@ -328,6 +344,31 @@ public function removeReferenceNumber()
     {
         $this->referenceNumber = null;
     }
+    
+    /**
+     * @return ReferenceNumber
+     */
+    public function getReferenceNumber2()
+    {
+        return $this->referenceNumber2;
+    }
+
+    /**
+     * @param ReferenceNumber $referenceNumber
+     *
+     * @return Package
+     */
+    public function setReferenceNumber2(ReferenceNumber $referenceNumber)
+    {
+        $this->referenceNumber2 = $referenceNumber;
+
+        return $this;
+    }
+
+    public function removeReferenceNumber2()
```

Comment:
Ah, I see. Both methods don't make a lot of sense indeed, but as it's the first one in there I think it's best to keep it together. Thanks :)